pkg/updater/version.go

```diff
@@ -61,7 +64,45 @@ type AppResponse struct {
 }
 
 type versionConfig struct {
-	ServerURL string `env:"ABC_UPDATER_URL,default=https://abc-updater.tycho.joonix.net"`
+	ServerURL      string   `env:"ABC_UPDATER_URL,default=https://abc-updater.tycho.joonix.net"`
+	IgnoreVersions []string `env:"IGNORE_VERSIONS"`
```

Comment:
Maybe `ABC_` prefix here too?

Reply:
That would technically be a breaking change in `abc`. If someone had previously set the `ABC_IGNORE_VERSIONS` env var, they would need to change it to `ABC_ABC_IGNORE_VERSIONS`.

Reply:
Ah, OK.